security/cert_manager_operator/cert-manager-operator-issuer-acme.adoc

```diff
@@ -20,5 +42,29 @@ include::modules/cert-manager-acme-dns-providers.adoc[leveloffset=+2]
 // Configuring an ACME issuer to solve HTTP01 challenges
 include::modules/cert-manager-acme-http01.adoc[leveloffset=+1]
 
-// Configuring an ACME issuer to solve DNS01 challenges
-include::modules/cert-manager-acme-dns01-aws.adoc[leveloffset=+1]
+// Configuring an ACME issuer to solve DNS01 challenges by using explicit credentials on AWS
+include::modules/cert-manager-acme-dns01-explicit-aws.adoc[leveloffset=+1]
+
+//Configuring an ACME issuer to solve DNS01 challenges by using ambient credentials on AWS
+include::modules/cert-manager-acme-dns01-ambient-aws.adoc[leveloffset=+1]
+
+// Configuring an ACME issuer to solve DNS01 challenges by using explicit credentials on GCP
+include::modules/cert-manager-acme-dns01-explicit-gcp.adoc[leveloffset=+1]
+
+// Configuring an ACME issuer to solve DNS01 challenges by using ambient credentials on GCP
+include::modules/cert-manager-acme-dns01-ambient-gcp.adoc[leveloffset=+1]
+
+// Configuring an ACME issuer to solve DNS01 challenges by using explicit credentials on Microsoft Azure
+include::modules/cert-manager-acme-dns01-explicit-azure.adoc[leveloffset=+1]
+
+[role="_additional-resources"]
+[id="additional-resources_cert-manager-operator-issuer-acme"]
+== Additional resources
```

Comment:
Something weird going on here; these links aren't working.

Looked at [published 4.13 cert-manager Operator docs](https://59702--docspreview.netlify.app/openshift-enterprise/latest/security/cert_manager_operator/index.html), and I see all of these chapters:
![image](https://github.com/openshift/openshift-docs/assets/61474374/5769d34c-18a2-4f2a-b24a-665d1ccaf64e)

But in the preview, all the AWS and GCP stuff is missing :thinking: 
![image](https://github.com/openshift/openshift-docs/assets/61474374/83f87222-29c9-4c63-8255-f187c8ec2b3a)

This PR doesn't change the topic map, so I am really not sure what's going on here.

Reply:
@jeana-redhat Rebasing and pushing worked. Now, the links are working and AWS/GCP sections are showing. Thanks you :)